sn.c

```diff
@@ -919,7 +923,11 @@ int main(int argc, char * const argv[]) {
 
   traceEvent(TRACE_NORMAL, "supernode started");
 
+#ifndef WIN32
   signal(SIGHUP, dump_registrations);
+#else
+  signal(SIGINT, dump_registrations);
```

Comment:
Please avoid handling the SIGINT signal, it should be used to terminate the program as normal